mobify/platform-scaffold
# Allow Debuggers In "Dev Mode"

Before, Uglify was using its default setting that removes debuggers from the compiled code. This is unexpected, since developers may want debuggers to assist them during troubleshooting. This shouldn't be an issue (i.e. accidentally committing debuggers) because of linters, but also because Uglify for production will remove debuggers from the final compiled bundle.

 **JIRA**: (link to JIRA ticket)
 **Linked PRs**: (links to corresponding PRs, optional)

## Changes
- Disable Uglify feature that removes debuggers, but only for the dev environment, not for production!

## How to test-drive this PR
- Run `npm start`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Add a debugger somewhere on the homepage container
- Refresh the page
- Verify that the debugger is caught by Chrome dev tools
